ts/session/utils/Menu.tsx

```diff
@@ -234,8 +234,13 @@ export function getDisappearingMenuItem(
       Boolean(isBlocked)
     )
   ) {
+    const isRtlMode = isRtlBody();
     return (
-      <SubMenu title={i18n('disappearingMessages') as any}>
+      // Remove the && false to make context menu work with RTL support
+      <SubMenu
+        title={i18n('disappearingMessages') as any}
+        rtl={isRtlMode && false}
```

Comment:
```suggestion
        rtl={isRtlMode}
```

Reply:
I voluntary keep it that way, until the whole UI is rtl ready.
If we want to add this, we need to make the whole UI mirrored.
Because enabling this, will make submenu appear on the left of the menu, but the arrow is still on the right and looking on the right.

Reply:
ah right